tests/main.sh

```diff
@@ -172,10 +176,27 @@ while getopts "hH?vAs:a:x:rl:p:S:" opt; do
 			PROVIDER="${CLOUD}"
 		fi
 		export BOOTSTRAP_PROVIDER="${PROVIDER}"
+		export BOOTSTRAP_CLOUD="${PROVIDER}"
 		;;
 	p)
 		export BOOTSTRAP_PROVIDER="${OPTARG}"
 		;;
+	c)
+	  PROVIDER=$(juju clouds --client --format=json | jq -r ".[\"${OPTARG}\"] | .type")
+	  export BOOTSTRAP_PROVIDER="${PROVIDER}"
+	  num_regions=$(juju clouds --client --format=json | jq -r ".[\"${OPTARG}\"] | .regions | length")
+	  if [[ ${num_regions} -gt 1 ]] ; then
+	    echo "more than 1 region, must specify"
+	    exit 1
+	  fi
+	  CLOUD="${OPTARG}"
+	  REGION=$(juju clouds --client --format=json | jq -r ".[\"${CLOUD}\"] | .regions | keys[0]")
+	  export BOOTSTRAP_REGION="${REGION}"
+	  export BOOTSTRAP_CLOUD="${CLOUD}"
+	  ;;
+	R)
+	  export BOOTSTRAP_REGION="${OPTARG}"
+	  ;;
```

Comment:
This is great, but can we fix the indentation, use `shfmt -l -s -w tests/**/*.sh` from the root of juju.

Reply:
My IDE is not cooperating, I was hoping there was a fmt command we could use!